### PR TITLE
Add IcalUtil

### PR DESCRIPTION
**Pull request by Riddarvid**

### CODE REVIEW

File: src/main/java/it/chalmers/eventItCal/utils/IcalUtil.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        for (Event event : events) {
+            VEvent vEvent = new VEvent(new DateTime(event.getStartTime()), new DateTime(event.getEndTime()), event.getName());
+            vEvent.getProperties().add(new Uid(event.getId().toString()));
+            vEvent.getProperties().add(new Description(event.getDescription()));
+            vEvent.getProperties().add(new Created(new DateTime(event.getCreateDate())));
+            try {
+                vEvent.getProperties().add(new Organizer(event.getOrganizer()));
+            } catch (URISyntaxException e) {
+                e.printStackTrace();
+            }
```
Comment (cajoho99) [inline src/main/java/it/chalmers/eventItCal/utils/IcalUtil.java:29]: We should check what an Organizer actually is in ical

File: src/main/java/it/chalmers/eventItCal/utils/IcalUtil.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            vEvent.getProperties().add(new Contact(event.getContact()));
+            calendar.getComponents().add(vEvent);
+        }
+        try {
+            FileOutputStream fout = new FileOutputStream("/home/riddarvid/digit/eventit-cal/src/main/java/it/chalmers/eventItCal/utils/meep.ics");
+            toFile(fout, calendar);
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+        }
```
Comment (cajoho99) [inline src/main/java/it/chalmers/eventItCal/utils/IcalUtil.java]: Exclude

File: src/main/java/it/chalmers/eventItCal/utils/IcalUtil.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        return calendar.toString();
+    }
+
+    public static void toFile(FileOutputStream fout, Calendar calendar) {
+        try {
+            CalendarOutputter outputter = new CalendarOutputter();
+            outputter.output(calendar, fout);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
```
Comment (cajoho99) [inline src/main/java/it/chalmers/eventItCal/utils/IcalUtil.java:42]: Might not be neccessary but could be good for debug

**Review by cajoho99 — APPROVED** [review]

**Review by Noxemia — APPROVED** [review]
LoOkS cOoL

**Pull request merged**